README.md

```diff
@@ -37,29 +39,72 @@ Install `Docker` according to the [official documentation](https://docs.docker.c
-| `email` | The email address you want to identify yourself in the project. | **email@example.com** |
-| `python_package_name` | Name of the Python package used to configure your project. It needs to be Python-importable, so no dashes, spaces or special characters are allowed. | **plone_site** |
-| `plone_version` | Plone version to be used. This queries for the latest available Plone 6 version and presents it to you as the default value. | **6.0.0a4** |
-| `volto_version` | Volto (Plone Frontend) version to be used. This queries for the latest available stable Volto version and presents it to you as the default value. | **16.0.0-alpha.8** |
-| `language_code` | Language to be used on the site. | **pt-br** |
-| `github_organization` | Used for GitHub and Docker repositories. | **collective**|
-| `container_registry` | Container registry to be used. | **Docker Hub**|
+| Option                | Description                                                                                                                                          | Example                       |
+| --------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------- |
+| `project_title`       | Your project's human-readable name, capitals and spaces allowed.                                                                                     | **Plone Site**                |
+| `project_slug`        | Your project's slug without spaces. Used to name your repository and Docker images.                                                                  | **plone-site**                |
+| `description`         | Describes your project and gets used in places like ``README.md`` and such.                                                                          | **New site for our company.** |
+| `author`              | This is you! The value goes into places like ``LICENSE``, ``setup.py`` and such.                                                                     | **Our Company**               |
+| `email`               | The email address you want to identify yourself in the project.                                                                                      | **email@example.com**         |
+| `python_package_name` | Name of the Python package used to configure your project. It needs to be Python-importable, so no dashes, spaces or special characters are allowed. | **plone_site**                |
+| `plone_version`       | Plone version to be used. This queries for the latest available Plone 6 version and presents it to you as the default value.                         | **6.0.0a4**                   |
+| `volto_version`       | Volto (Plone Frontend) version to be used. This queries for the latest available stable Volto version and presents it to you as the default value.   | **16.0.0-alpha.8**            |
```

Comment:
The example versions here are already outdated and it is difficult to keep it always up to date. Anyway, even if complaining, I have no idea how to deal with it.

Reply:
Well, can we fix this when a fairy sends us some extra hours?
![grafik](https://user-images.githubusercontent.com/1144737/191926163-4461d123-bbb2-4ca7-9614-3ecabb92bbf0.png)